Updates to files for page speed issues

oscarShift4 opened the pull request:
Updates to files for page speed issues

oscarShift4 committed:
Updates to files for page speed issues

Updates to files for page speed issues

ShevonJ reviewed (changes requested):
See revisions on FE-1338 - https://3dcart.atlassian.net/browse/FE-1338

oscarShift4 committed:
last updates made

Updates from comments in the FE

oscarShift4 committed:
fontello duplicate

fixed

oscarShift4 committed:
Update requested by Atif end of the day

I try Adding different "arias" and could not get to the right one. I try others and it would give me lower score instead.
